installing/installing_bare_metal/installing-bare-metal.adoc

```diff
@@ -138,10 +138,9 @@ include::modules/installation-bare-metal-config-yaml.adoc[leveloffset=+2]
 .Additional resources
 
 * See xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installation-load-balancing-user-infra_installing-bare-metal[Load balancing requirements for user-provisioned infrastructure] for more information on the API and application ingress load balancing requirements.
-* See xref:../../post_installation_configuration/enabling-cluster-capabilities.adoc[Enabling cluster capabilities] for more information on enabling cluster capabilities that were disabled prior to installation.
+* See xref:../../installing/overview/cluster-capabilities.adoc#cluster-capabilities[Cluster capabilities] for more information on enabling cluster capabilities that were disabled before installation.
```

Comment:
I will leave these as they are for now because changing them would make them inconsistent with the format of the other links in the assembly, and changing them all would be out of scope for this PR